File: 백준/Bronze/1978. 소수 찾기/소수 찾기.c
#include <stdio.h>

int main(void) {  
   int a;
   scanf("%d", &a);

   int arr[a], result = 0, count;

   for(int i = 0; i < a; i++){
      scanf("%d", &arr[i]);
   }

   for(int i = 0; i < a; i++){
      count = 0;
      for(int j = 1; j < arr[i] + 1; j++){
         if(arr[i] % j == 0){
            count += 1;
         }
      }
      if(count == 2){
         result += 1;
      }
   }
   
   printf("%d", result);

   return 0;
}

File: 백준/Bronze/1978. 소수 찾기/소수 찾기.py
a = int(input())
arr = list(input().split())

result = 0
for i in range(0, a):
  count = 0
  for j in range(1, int(arr[i]) + 1):
    if int(arr[i]) % j == 0:
      count += 1
  
  if count == 2:
    result += 1

print(result)